lib/plugins/helper/open_graph.js

```diff
@@ -59,12 +58,16 @@ function openGraphHelper(options = {}) {
   if (!images.length && content) {
```

Comment:
Is `if (!images.length && content && content.includes('<img'))` safe? I'm wondering if `content` can be null. Let me try with an empty file.

Reply:
Maybe some syntax error might cause content to be null, I'm not sure.